client/state/sites/vouchers/actions.js

```diff
@@ -103,3 +113,33 @@ export function requestSiteVouchers( siteId ) {
 			} );
 	};
 }
+
+/**
+ * Assign a voucher to the given site.
+ *
+ * @param {Number} siteId - identifier of the site
+ * @param {String} serviceType - service type
+ * @returns {Function} a promise that will resolve once fetching is completed
+ */
+export function assignSiteVoucher( siteId, serviceType ) {
+	return dispatch => {
+		dispatch( vouchersRequestAction( siteId ) );
+
+		return wpcom
+			.site( siteId )
+			.adCreditVouchers()
+			.assign( serviceType )
```

Comment:
Are these works-in-progress? Were they meant to be `.then( adCreditVouchers )` or something like that? Also above in `requestSiteVouchers`

By the way, do these functions actually exist yet? I couldn't find them, despite the above code (`requestSiteVouchers()`) already being checked in to **master**